src/Ed25519VerificationKey2018.js

```diff
@@ -145,30 +145,33 @@ class Ed25519VerificationKey2018 extends LDVerifierKeyPair {
   }
 
   /**
-   * Adds a public key base to a public key node.
+   * Exports the serialized representation of the KeyPair
+   * and other information that json-ld Signatures can use to form a proof.
    *
-   * @param {object} key - The public key object in a jsonld-signature.
-   * @param {string} key.publicKeyBase58 - Base58btc encoded Public Key.
+   * @param {object} [options={}] - Options hashmap.
```

Comment:
this does not match the API definition.

Reply:
Thanks, fixing.

Reply:
Fixed (updated the function to match comment)